### ISSUE TITLE
wallabag non trouvé à cette adresse -- Vérifiez l'URL

### ISSUE THREAD

**Issue by disque-monde** [wallabag/wallabag#3788]
Wallabag android app error after upgrade to 2.3.4. The URL is correct. Acces in firefox is nice.

apache acces.log :
```
83.xxx.xxx.xxx - - [26/Nov/2018:17:19:34 +0100] "GET / HTTP/1.1" 302 4260 "-" "okhttp/3.8.1"
83.xxx.xxx.xxx - - [26/Nov/2018:17:19:34 +0100] "GET /login HTTP/1.1" 200 1535 "-" "okhttp/3.8.1"
83.xxx.xxx.xxx - - [26/Nov/2018:17:19:35 +0100] "GET / HTTP/1.1" 302 4260 "-" "okhttp/3.8.1"
83.xxx.xxx.xxx - - [26/Nov/2018:17:19:35 +0100] "GET /login HTTP/1.1" 200 1536 "-" "okhttp/3.8.1"
83.xxx.xxx.xxx - - [26/Nov/2018:17:19:35 +0100] "GET /login/ HTTP/1.1" 404 4246 "-" "okhttp/3.8.1"

```

**Comment by giuppi** [comment]
already fixed in 2.3.5 upcoming release.

**Comment by Kdecherf** [comment]
@giuppi is there a workaround available for this issue pending the upcoming release?

**Comment by giuppi** [comment]
https://github.com/wallabag/wallabag/commit/6f2287da69841b98c4c0db93b6dce1325ba4311d

**Comment by Kdecherf** [comment]
@giuppi does the android app works without it or is it blocking?

**Comment by giuppi** [comment]
with the modified file `src/Wallabag/UserBundle/Resources/views/layout.html.twig` the app is able to connect to the server and fetch data.
clear cache after replacing it

**Comment by Kdecherf** [comment]
Fixed by https://github.com/wallabag/wallabag/pull/3784
Will be released as wallabag 2.3.5
